app.js

```diff
@@ -134,6 +99,22 @@ var server = http.createServer(function(request, response) {
         defaultResult(response, query, now);
     }
 });
+var waitForElastic = process.argv[2] == 'force-elastic';
 
-console.log('Start listening %d port', config.env.port);
-server.listen(config.env.port); 
+var is_elastic_enabled = false;
```

Comment:
It's better to move `is_elastic_enabled` before `defaultResult`, because this variable is used by the function, and when somebody read file from top to bottom, nobody will understand, where is is_elastic_enabled variable defined.